src/main/java/com/vinylteam/vinyl/util/impl/VinylUaParser.java

```diff
@@ -15,57 +17,99 @@
 import java.util.List;
 
 public class VinylUaParser implements VinylParser {
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
     private final String startLink = "http://vinyl.ua";
+    private final String classContainingGenresLinks = "dropdown-menu dropdown-menu-left";
```

Comment:
I thought that we don't use genres in project. Perhaps I mistake?

Reply:
It's to get genre links in VinylsUaParser